qutebrowser/app.py

```diff
@@ -210,14 +210,17 @@ def _load_session(name):
         name: The name of the session to load, or None to read state file.
     """
     state_config = objreg.get('state-config')
+    session_manager = objreg.get('session-manager')
     if name is None:
         try:
             name = state_config['general']['session']
         except KeyError:
             # No session given as argument and none in the session file ->
             # start without loading a session
-            return
-    session_manager = objreg.get('session-manager')
+            if session_manager.exists('_autosave'):
```

Comment:
What's the rationale for that? I can't really see the point - if I do `qutebrowser -r foo` and use it for an hour (but never save my session explicitly) and it crashes, surely I'd want the autosaved sessions to be loaded, no?